cluster/log-dump/log-dump.sh

```diff
@@ -225,22 +226,22 @@ function save-logs-windows() {
     local -r node="${1}"
     local -r dest_dir="${2}"
 
-    if [[ ! "${gcloud_supported_providers}" =~ "${KUBERNETES_PROVIDER}" ]]; then
+    if [[ ! "${gcloud_supported_providers}" =~ ${KUBERNETES_PROVIDER} ]]; then
       echo "Not saving logs for ${node}, Windows log dumping requires gcloud support"
       return
     fi
 
     export-windows-docker-event-log "${node}"
 
     local remote_files=()
-    for file in ${windows_node_logfiles[@]}; do
+    for file in ${windows_node_logfiles}; do
```

Comment:
shellcheck detects SC2068.
```
In ./cluster/log-dump/log-dump.sh line 236:
    for file in ${windows_node_logfiles[@]}; do
                ^-------------------------^ SC2068: Double quote array expansions to avoid re-splitting elements.
```

Reply:
er, why didn't we add quotes then??
https://github.com/koalaman/shellcheck/wiki/SC2068